packages/modelviewer.dev/data/docs.json

```diff
@@ -5,65 +5,65 @@
       {
         "name": "src",
         "htmlName": "src",
-        "description": "<p>The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.</p>",
+        "description": "The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.",
         "default": [
           "String",
-          "any legal filepath",
-          "empty string"
+          "empty string",
+          "any legal filepath"
         ]
       },
       {
         "name": "alt",
         "htmlName": "alt",
-        "description": "<p>Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.</p>",
+        "description": "Configures the model with custom text that will be used to describe the model to viewers who use a screen reader or otherwise depend on additional semantic context to understand what they are viewing.",
         "default": [
           "String",
-          "any description",
-          "empty string"
+          "empty string",
+          "any description"
         ]
       },
       {
         "name": "poster",
         "htmlName": "poster",
-        "description": "<p>Displays an image instead of the model, useful for showing the user something before a model is loaded and ready to render. If you use a poster with transparency, you may also want to set --poster-color to transparent so that the background shows through.</p>",
+        "description": "Displays an image instead of the model, useful for showing the user something before a model is loaded and ready to render. If you use a poster with transparency, you may also want to set --poster-color to transparent so that the background shows through.",
```

Comment:
A little picky, but can you keep the sidebar text from reflowing when it animations in and out? Also, didn't the content use to scroll to position when you clicked a sidebar link? It just jumps there now, but the scroll was kind of nice. Maybe you had a good reason though?

Reply:
theres a few reasons for that: given that we aren't using any frameworks, it would have involved a lot of finagling to do the auto-jump when we are arriving from an external link vs. scrolling when you are within a page, additionally it also saves quite a lot of time when you jumping through different attributes.

and yeah, I'll look into the sidebar text

Reply:
done